includes/classes/Product.php

```diff
@@ -90,7 +89,7 @@ public function getId(): ?int
 
     public function exists(): bool
     {
-        return !empty($this->product_id);
```

Comment:
Parentheses not needed.

Instead of checking deprecated EOF property, we can check what its set from: `empty($this->data)`